ubireader/ubifs/output.py

```diff
@@ -112,7 +114,9 @@ def extract_dents(ubifs, inodes, dent_node, path='', perms=False):
     elif dent_node.type == UBIFS_ITYPE_LNK:
         try:
             # probably will need to decompress ino data if > UBIFS_MIN_COMPR_LEN
-            os.symlink('%s' % inode['ino'].data.decode('utf-8'), dent_path)
+            inode = inodes[dent_node.inum]
+            lnkname = decrypt_symlnk_target(inodes, dent_node)
```

Comment:
would be cleaner to do decryption during parsing rather than during post-processing

Reply:
Yeah but unfortunately you do not have all the inodes for key derivation if we walk through the nodes this way. Maybe we need refactoring.